tests/test_annotations.py

```diff
@@ -31,14 +31,16 @@ class C:
             y = attr.ib(type=str)
             z = attr.ib()
 
+        attr.resolve_types(C)
```

Comment:
You can also use resolve_types as a decorator which might make it easier to read.  (But you can't use the decorator if there are forward references)